### PR TITLE
chore: changelog v50.3.0

### PR DESCRIPTION
**Pull request by lcampos**
### What does this PR do?
Changelog for release v50.3.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# 50.3.0 - October 28, 2020
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- New force:apex:log:list command implementation ([PR #2644](https://github.com/forcedotcom/salesforcedx-vscode/pull/2644))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: @ekapner this command is no directly surfaced in the command palette, it is part of another command (Get logs). What do you think, should we leave it as is or reference where it's used ?

Comment (ekapner) [reply packages/salesforcedx-vscode/CHANGELOG.md:7]: @lcampos - I like the idea of providing more information.

Comment (lcampos) [reply packages/salesforcedx-vscode/CHANGELOG.md:7]: how about `New force:apex:log:list command implementation, used as part of 'SFDX: Get Apex Debug Logs ...'` ?

Comment (ekapner) [reply packages/salesforcedx-vscode/CHANGELOG.md]: That works, thanks

**Review by ekapner — APPROVED** [review]

**Pull request merged**